app/templates/spa.html

```diff
@@ -71,7 +71,8 @@
 		{% endif %}
 		
 		{# HACK CACHING : https://stackoverflow.com/questions/32414/how-can-i-force-clients-to-refresh-javascript-files #}
-		<script defer src="/static/build/bundle.js?v3"></script>
+		<script defer src="/static/build/bundle.js?v4"></script>
```

Comment:
Si je me souviens bien, ceci permet simplement de rafraichir le cache et n'est pas obligatoire,  correct ? Je suggeres donc de passer a la v5 pour faire "le mec" 
```suggestion
		<script defer src="/static/build/bundle.js?v5"></script>
```

Reply:
@JulienParis